templates/base/scripts/probe_thread_dump.sh

```diff
@@ -79,7 +79,14 @@ function main {
 	then
 		echo -e "${curl_content}"
 
-		kill -3 $(pgrep -f "org.apache.catalina.startup.Bootstrap")
+		out=$(jattach $(ps -ef | grep org.apache.catalina.startup.Bootstrap | grep -v grep | awk '{print $1}') threaddump)
+		
+		if [[ ! -d $LIFERAY_THREAD_DUMP_DIRECTORY ]]
+		then
+			mkdir -R $LIFERAY_THREAD_DUMP_DIRECTORY
+		fi
+
+		echo -e "${out}" > "${LIFERAY_THREAD_DUMP_DIRECTORY}/$(hostname)_$(date +"%Y-%m-%dT%H:%M:%S%z").tdump"
```

Comment:
Nem lenne jó ha ki is írnánk ide is? Talán sima docker esetén lenne haszna, vagy nem?

Reply:
De, sima docker esetében jól jöhet. Beteszek egy sia echo-t.